cmd/kops/create_keypair.go

```diff
@@ -268,7 +268,9 @@ func createKeypair(out io.Writer, options *CreateKeypairOptions, name string, ke
 }
 
 func completeKeyset(cluster *kopsapi.Cluster, clientSet simple.Clientset, args []string, filter func(name string, keyset *fi.Keyset) bool) (keyset *fi.Keyset, keyStore fi.CAStore, completions []string, directive cobra.ShellCompDirective) {
-	keyStore, err := clientSet.KeyStore(cluster)
+	ctx := context.TODO()
```

Comment:
I don't think this qualifies for TODO. This is a new interactive context.

Reply:
So I've been using TODO at the "frontier" to limit how far we have to require context.Context in the callchains.

However, in most of these we do have the cobra Command available, which includes the context.Context, so I added another commit to add those to this PR.